resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -74,6 +75,9 @@ class KubernetesSuite extends SparkFunSuite
 
   protected override def logForFailedTest(): Unit = {
     logInfo("\n\n===== EXTRA LOGS FOR THE FAILED TEST\n")
+    logInfo("BEGIN driver DESCRIBE POD\n" +
+      Minikube.describePods(s"spark-app-locator=$appLocator,spark-role=driver").mkString("\n"))
```

Comment:
Before merging this I intend to remove the `spark-role=driver` label and change the section from `BEGIN driver DESCRIBE POD` to `BEGIN DESCRIBE POD for application` (and update the `END` too accordingly).

To include the executors too.

Reply:
+1 for the idea.